### PR TITLE
test(e2e): improve test case for line shifting and style with more colors

### PR DESCRIPTION
**Pull request by asturur**
## Motivation

Improve the test case with more colors and deletion to showcase the text issues that needs to be fixed in #9197 

I extended a the ObjectUtil class creating TextUtil class that contains a method to click the text at a specific point rather than guessing where.

### CODE REVIEW

File: e2e/tests/test.ts [hunk 1]
```diff
@@ -13,6 +14,9 @@ type AsyncReturnValue<T> = T | Promise<T>;
 const setupTasks: Promise<void>[] = [];
 const teardownTasks: Awaited<VoidFunction>[] = [];
 
+// makes possible call things in browser context.
+window.fabric = fabric;
```
Comment (asturur) [inline e2e/tests/test.ts:18]: not going to do anything fancier than this. this works

**Comment by asturur** [comment]
nothing to see here, improved a test.

File: e2e/tests/test.ts (hunk 1 above)
Comment (ShaMan123) [reply e2e/tests/test.ts:18]: I am interestred where it is used

File: e2e/utils/TextUtil.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import type { IText, Point } from 'fabric';
+
+export class TextUtil<T extends IText = IText> extends ObjectUtil<T> {
+  async getCanvasCursorPositionAt(index: number): Promise<Point> {
+    return this.executeInBrowser(
+      (object, { index }) => {
+        const { left, top, topOffset, leftOffset } =
+          object._getCursorBoundaries(index, true);
+        // add extra 10 pixels to avoid controls
+        const point = new fabric.Point(
```
Comment (asturur) [inline e2e/utils/TextUtil.ts:11]: @ShaMan123 here i needed to use it.
I can't import Point here nor pass it as a context in executeInBrowser

Comment (ShaMan123) [reply e2e/utils/TextUtil.ts:11]: I see, thanks

**Pull request merged**